DotaStats/Data/MatchInfoProcessing/MatchDetailToEventsConverter.swift

```diff
@@ -0,0 +1,148 @@
+import Foundation
+
+class MatchDetailToEventsConverter {
+    typealias ConvertedEventsDictionary = [Int: [MatchEvent]]
+
+    static func convert(_ details: MatchDetail) -> ConvertedEventsDictionary {
+        var dictionaryToReturn: ConvertedEventsDictionary = [:]
```

Comment:
Можно так и назвать `convertedEventsDictionary`, тогда будет попонятнее
```suggestion
        var convertedEventsDictionary: ConvertedEventsDictionary = [:]
```